modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -95,257 +95,257 @@ else {
+	});
+}
+
+function <portlet:namespace />doDeleteOrganizations(
+	organizationIds,
+	organizationsRedirect
+) {
+	var form = document.<portlet:namespace />fm;
+
+	if (organizationsRedirect) {
+		Liferay.Util.setFormValues(form, {
+			redirect: organizationsRedirect,
 		});
 	}
 
-	function <portlet:namespace />doDeleteOrganizations(
-		organizationIds,
-		organizationsRedirect
-	) {
-		var form = document.<portlet:namespace />fm;
+	Liferay.Util.postForm(form, {
+		data: {
+			deleteOrganizationIds: organizationIds,
+			<%= Constants.CMD %>: '<%= Constants.DELETE %>',
+		},
+		url: '<portlet:actionURL name="/users_admin/edit_organization" />',
+	});
+}
 
-		if (organizationsRedirect) {
-			Liferay.Util.setFormValues(form, {
-				redirect: organizationsRedirect,
-			});
-		}
+function <portlet:namespace />getUsersCount(className, ids, status, callback) {
+	var formData = new FormData();
 
-		Liferay.Util.postForm(form, {
-			data: {
-				deleteOrganizationIds: organizationIds,
-				<%= Constants.CMD %>: '<%= Constants.DELETE %>',
-			},
-			url: '<portlet:actionURL name="/users_admin/edit_organization" />',
+	formData.append('className', className);
+	formData.append('ids', ids);
+	formData.append('status', status);
+
+	Liferay.Util.fetch(
+		'<liferay-portlet:resourceURL id="/users_admin/get_users_count" />',
+		{
+			body: formData,
+			method: 'POST',
+		}
+	)
+		.then(function (response) {
+			return response.text();
+		})
+		.then(function (response) {
+			callback(response);
 		});
-	}
+}
 
-	function <portlet:namespace />getUsersCount(className, ids, status, callback) {
-		var formData = new FormData();
-
-		formData.append('className', className);
-		formData.append('ids', ids);
-		formData.append('status', status);
-
-		Liferay.Util.fetch(
-			'<liferay-portlet:resourceURL id="/users_admin/get_users_count" />',
-			{
-				body: formData,
-				method: 'POST',
-			}
-		)
-			.then(function (response) {
-				return response.text();
-			})
-			.then(function (response) {
-				callback(response);
-			});
-	}
+function <portlet:namespace />showUsers(status) {
 
-	function <portlet:namespace />showUsers(status) {
+	<%
+	PortletURL showUsersURL = renderResponse.createRenderURL();
 
-		<%
-		PortletURL showUsersURL = renderResponse.createRenderURL();
+	showUsersURL.setParameter("mvcRenderCommandName", "/users_admin/view");
+	showUsersURL.setParameter("toolbarItem", toolbarItem);
+	showUsersURL.setParameter("usersListView", usersListView);
 
-		showUsersURL.setParameter("mvcRenderCommandName", "/users_admin/view");
-		showUsersURL.setParameter("toolbarItem", toolbarItem);
-		showUsersURL.setParameter("usersListView", usersListView);
+	organizationId = ParamUtil.getLong(request, "organizationId", OrganizationConstants.DEFAULT_PARENT_ORGANIZATION_ID);
 
-		organizationId = ParamUtil.getLong(request, "organizationId", OrganizationConstants.DEFAULT_PARENT_ORGANIZATION_ID);
+	if (organizationId != OrganizationConstants.DEFAULT_PARENT_ORGANIZATION_ID) {
```

Comment:
Ha, I was going to suggest `!==` here and then realized it was Java. 😂

There *aren* some `==` elsewhere in this PR that would be better changed to `===`; every little step towards being able to turn on the `eqeqeq` ESLint rule helps... Up to you whether you want to do it or not (totally optional!).

Reply:
Thanks for the heads up - I'm going to replace those